packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -52,6 +53,11 @@ interface Point {
   y: number;
 }
 
+interface InspectorUnavailableBoxData {
+  visible: boolean;
```

Comment:
`visible` is only ever set to `true`. I'd change it to get rid of this interface and instead just have 
```tsx
  const [inspectorUnavailablePosition, setInspectorUnavailablePosition] = useState<Point | null>(null);
```